code/game/machinery/gulag_teleporter.dm

```diff
@@ -21,9 +21,17 @@ The console is located at computer/gulag_teleporter.dm
 	var/jumpsuit_type = /obj/item/clothing/under/rank/prisoner
 	var/shoes_type = /obj/item/clothing/shoes/sneakers/orange
 	var/obj/machinery/gulag_item_reclaimer/linked_reclaimer = null
+	var/list/required_items
 
 /obj/machinery/gulag_teleporter/New()
 	..()
+	required_items = typecacheof(list(
+				/obj/item/weapon/implant,
+				/obj/item/clothing/suit/space/eva/plasmaman,
+				/obj/item/clothing/under/plasmaman,
```

Comment:
Yes see the if check below that I referenced. It removes items in the if. (The other bug was it removing items by unequipping without it going to the reclaimer. Literally helemt and such popping out the machine. It was solved by putting the plasmaman check before the unequip. Ask @Antur for details)